kataras/iris
# Add option DisplayFullBanner to display one-line banner

### Summary

**- Introduction**

A big thanks to @kataras for this amazing tool! 🍻 

**- What I did**

I add a new option `DisplayFullBanner` to display a small banner (one line).
It's very useful because we do an economy of ~7 lines in logs.

**- How I did it**

I've add the option `DisplayFullBanner` (default `true`).

**- How to verify it**

Create a small project with this option set to `false`:
```go
package main

import (
    "github.com/kataras/iris"
)

func main() {
        iris.Get("/iris-is-awesome", func(ctx *iris.Context){})
	iris.Config.DisplayFullBanner = false
	iris.Listen("8080")
}
```

And you should see:

`Sat, 28 Jan 2017 15:11:00 GMT: Iris x.x.x - Running at 0.0.0.0:8000`

### Changelog

#### Description

Add `DisplayFullBanner` option to display one-line banner at startup Iris.

#### Questions

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests?        | no
